build/zig/build-0.11.sh

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/bash
+#
+# {{{ CDDL HEADER
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"), version 1.0.
+# You may only use this file in accordance with the terms of version
+# 1.0 of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+# }}}
+
+# Copyright 2022 OmniOS Community Edition (OmniOSce) Association.
+
+. ../../lib/build.sh
+
+PROG=zig
+VER=0.11.0
+MAJVER=0
+MINVER=11
+PKG=ooce/developer/zig-0.11
```

Comment:
We usually don't use dots for versions in package FMRIs. Could you please change this to `zig-011`.

Reply:
Done.